addons/im_livechat/models/res_partner.py

```diff
@@ -55,6 +58,17 @@ def _compute_user_livechat_username(self):
         for partner in self:
             partner.user_livechat_username = next(iter(partner.user_ids.mapped('livechat_username')), False)
 
+    def _compute_livechat_channel_count(self):
+        channel_count_dict = dict(
+            self.env["im_livechat.channel.member.history"]._read_group(
+                domain=[("partner_id", "in", self.ids)],
+                groupby=["partner_id"],
+                aggregates=["__count"],
```

Comment:
Small issue. If the partner leaves/joins several time, the count will be incorrect as member history will have several lines for the same channel. We should count distinct channel ids.
```suggestion
                aggregates=["channel_id:count_distinct"],
```